tests/e2e/util.go

```diff
@@ -2599,3 +2540,285 @@ func getRestConfigClient() *rest.Config {
 	}
 	return restConfig
 }
+
+// simulatePasswordRotation invokes password roation happens immediately
+// on the given vCenter host over SSH, vmon-cli is used to restart the wcp service after changing the time
+func simulatePasswordRotation(file, host string) error {
+	sshCmd := fmt.Sprintf("sed -i '3 c\\0' %s", file)
+	framework.Logf("Invoking command %v on vCenter host %v", sshCmd, host)
+
+	result, err := fssh.SSH(sshCmd, host, framework.TestContext.Provider)
+	if err != nil || result.Code != 0 {
+		framework.Logf("Result : %s", result)
+		return fmt.Errorf("couldn't execute command: %s on vCenter host: %v", sshCmd, err)
+	}
+
+	sshCmd = fmt.Sprintf("vmon-cli -r %s", wcpServiceName)
+	framework.Logf("Invoking command %v on vCenter host %v", sshCmd, host)
+	result, err = fssh.SSH(sshCmd, host, framework.TestContext.Provider)
+	ginkgo.By("Waiting for 30 seconds for wcp service to restart and perform password rotation")
+	time.Sleep(sleepTimeOut)
+	if err != nil || result.Code != 0 {
+		framework.Logf("Result : %s", result)
+		return fmt.Errorf("couldn't execute command: %s on vCenter host: %v", sshCmd, err)
+	}
+	return nil
+}
+
+// Scale scales ss to count replicas.
+func scale(c clientset.Interface, ss *appsv1.StatefulSet, count int32) (*appsv1.StatefulSet, error) {
```

Comment:
May be add 'sts' to method names to indicate it is related to statefulset.
Also, add a comment to mention that we are using customized methods from framework due to uncustomizable timeouts in framework.

Reply:
Done